.evergreen/.evg.yml

```diff
@@ -0,0 +1,46 @@
+functions:
```

Comment:
Rename this file to config.yml.  Java driver file name is not what most newer projects use.